### PR TITLE
Hfeyp 560 cms mobile view

### PR DESCRIPTION
**Pull request by tunde-anddigital**
## Ticket and context

Ticket: [HFEYP-560](https://dfedigital.atlassian.net/browse/HFEYP-560)
## Tech review
- Added correct mobile navigation menu to cms view (as per this link [Header](https://design-system.service.gov.uk/components/header/))
- Fixed issue with nav links disappearing at 200%+ zoom or when shrinking the page width

### How can someone see it in review app?
1. Click the link to review app (posted by a `github-actions` bot below)
2. Follow the steps from the ticket.

### CODE REVIEW

**Review by ebrett — APPROVED** [review]

File: app/views/layouts/cms.html.erb [hunk 1]
```diff
@@ -24,43 +24,46 @@
...
-                <image src="<%= asset_pack_path('media/images/govuk-logotype-crown.png') %>" class="govuk-header__logotype-crown-fallback-image"></image>
-              </svg>
-              <span class="govuk-header__logotype-text">
-                GOV.UK
-              </span>
+    <a href="#main-content" class="govuk-skip-link">Skip to main content</a>
```
Comment (reggieb) [inline app/views/layouts/cms.html.erb:30]: The content from lines 30 to 64 is within the `header` tag, so should be indented by two more spaces.

File: app/views/layouts/cms.html.erb [hunk 2]
```diff
@@ -24,43 +24,46 @@
       document.body.className = ((document.body.className) ? document.body.className + ' js-enabled' : 'js-enabled');
     </script>
 
+
+
```
Comment (reggieb) [inline app/views/layouts/cms.html.erb:28]: Are these two empty lines really necessary?

**Review by reggieb — COMMENTED** [review]
Looks good. A couple of very minor comments added

**Review by reggieb — APPROVED** [review]
LGTM

File: app/views/layouts/cms.html.erb (hunk 1 above)
Comment (reggieb) [reply app/views/layouts/cms.html.erb]: That has had the added benefit of reducing the difference between this branch and master - so made the changes clearer. Nice :smile:

**Pull request merged**